README.md

```diff
@@ -1,6 +1,6 @@
 # Keep It Breezy
 
-![Success Baby Poster](https://turingschool.slack.com/files/U0327VCBG59/F03A8AFM4CR/screen_shot_2022-04-03_at_8.48.38_am.png "Success Baby Poster")
+![Success Baby Poster](readme-imgs/SuccessBaby.png)
 
 ## Table of Contents [create an anchor](#anchors-in-markdown)
```

Comment:
Can you remove the [create an anchor](#anchors-in-markdown)?

Reply:
Can do!